data/mods/Aftershock/items/gun/laser.json

```
@@ -107,16 +107,175 @@
+    "price": "500 USD",
+    "price_postapoc": "500 USD",
+    "ammo": [ "battery" ],
+    "ranged_damage": { "damage_type": "heat", "amount": 5 },
+    "range": 15,
+    "modes": [ [ "DEFAULT", "Pulse ", 1 ], [ "BURST", "2s Sequence ", 2 ], [ "AUTO", "3s Sequence ", 3 ] ],
+    "delete": { "flags": [ "NO_UNLOAD", "NO_RELOAD" ] },
+    "ammo_to_fire": 10,
+    "ups_charges": 0,
+    "valid_mod_locations": [
+      [ "accessories", 2 ],
+      [ "emitter", 1 ],
+      [ "grip", 1 ],
+      [ "lens", 1 ],
+      [ "rail", 1 ],
+      [ "sights", 1 ],
+      [ "stock", 1 ],
+      [ "underbarrel", 1 ]
+    ],
+    "default_mods": [ "red_dot_sight" ],
+    "pocket_data": [
+      {
+        "pocket_type": "MAGAZINE_WELL",
+        "holster": true,
+        "max_contains_volume": "20 L",
+        "max_contains_weight": "20 kg",
+        "magazine_well": "75 ml",
+        "item_restriction": [ "afs_cartridge", "afs_compact_cartridge", "afs_bootleg_cartridge" ]
+      }
+    ]
+  },
+  {
+    "id": "afs_archeotech_laspistol",
+    "type": "GUN",
+    "name": { "str": "E-150b" },
+    "copy-from": "v29",
+    "description": "A silver, elegantly wrought laser pistol that fires armor-piercing bolts of golden light.  A now irreplaceable historical artifact built by the engineering AIs of the late Hyperspace Era, it is as precious as it is deadly.",
+    "price": "500 kUSD",
+    "price_postapoc": "500 kUSD",
+    "ammo": [ "battery" ],
+    "weight": "380 g",
+    "volume": "750 ml",
+    "ranged_damage": [ { "damage_type": "pure", "amount": 5 }, { "damage_type": "heat", "amount": 20 } ],
+    "range": 20,
+    "modes": [ [ "DEFAULT", "Pulse ", 1 ], [ "BURST", "2s Sequence ", 2 ] ],
+    "ammo_to_fire": 30,
+    "ammo_effects": [ "INCENDIARY", "DAZZLE_BEAM", "PLASMA_BUBBLE" ],
+    "delete": { "flags": [ "NO_UNLOAD", "NO_RELOAD" ] },
+    "ups_charges": 0,
+    "built_in_mods": [ "afs_holo_aim_assist" ],
+    "valid_mod_locations": [
+      [ "accessories", 2 ],
+      [ "emitter", 1 ],
+      [ "grip", 1 ],
+      [ "lens", 1 ],
+      [ "rail", 1 ],
+      [ "sights", 1 ],
+      [ "stock", 1 ],
+      [ "underbarrel", 1 ]
+    ],
+    "pocket_data": [
+      {
+        "pocket_type": "MAGAZINE_WELL",
+        "holster": true,
+        "max_contains_volume": "20 L",
+        "max_contains_weight": "20 kg",
+        "magazine_well": "75 ml",
+        "item_restriction": [ "afs_cartridge", "afs_compact_cartridge", "afs_bootleg_cartridge" ]
+      }
+    ]
+  },
+  {
+    "id": "afs_v29",
+    "type": "GUN",
+    "name": { "str": "V14 laser pistol" },
+    "copy-from": "v29",
+    "description": "An incredibly ancient and outdated laser pistol design whose origins can be traced to Earth's early Space Age.  One the few 'open-hardware' guns, its leaked and widely circulated design specifications allow this weapon to retain some popularity within backwater worlds.",
```

```suggestion
    "description": "An incredibly ancient and outdated laser pistol design whose origins can be traced to Earth's early Space Age.  One of the few 'open-hardware' guns, its leaked and widely circulated design specifications allow this weapon to retain some popularity within backwater worlds.",
```

---

```suggestion
    "description": "An incredibly ancient and outdated laser pistol design with origins traceable to Earth's early Space Age.  One of the few 'open-hardware' guns, its leaked and widely-circulated design specifications allow this weapon to retain some popularity within backwater worlds.",
```
Not sure about "leaked and widely-circulated" - "leaked then widely-circulated"?